src/static/css/hint.css

```diff
@@ -0,0 +1,40 @@
+@import url('../themes/default/content.css');
+@import url('../themes/dark/content.css');
+@import url('../themes/greenmat/content.css');
+
+span.TridactylHint {
+    position: absolute !important;
+    font-family: var(--tridactyl-hintspan-font-family) !important;
+    font-size: var(--tridactyl-hintspan-font-size) !important;
+    font-weight: var(--tridactyl-hintspan-font-weight) !important;
+    text-transform: uppercase !important;
+    color: var(--tridactyl-hintspan-fg) !important;
+    background-color: var(--tridactyl-hintspan-bg) !important;
+    border-color: var(--tridactyl-hintspan-border-color) !important;
+    border-width: var(--tridactyl-hintspan-border-width) !important;
+    min-width: .75em !important;
+    border-style: var(--tridactyl-hintspan-border-style) !important;
+    padding: 0 1pt !important;
+    text-align: center !important;
+    z-index: 2147483647 !important;
+}
```

Comment:
I feel like you've overdone it a bit here with the variables. I don't think that costs us much, but do remember that it's ok for a theme to just style these classes directly: we don't need and don't really want variables for everything, just for the most likely to change things as a convenience and for any value that's repeated in multiple places.